README.md

```diff
@@ -178,21 +145,30 @@ python -c "import onnx"
 to verify it works.
 
 
-#### Common Errors
-**Environment variables**: `USE_MSVC_STATIC_RUNTIME` (should be 1 or 0, not ON or OFF)
+## Common Build Options
+For full list refer to CMakeLists.txt  
+**Environment variables**  
+* `USE_MSVC_STATIC_RUNTIME` should be 1 or 0, not ON or OFF. When set to 1 onnx links statically to runtime library.  
+**Default**: USE_MSVC_STATIC_RUNTIME=1
 
-**CMake variables**: `ONNX_USE_PROTOBUF_SHARED_LIBS`, `Protobuf_USE_STATIC_LIBS`
+* `DEBUG` should be 0 or 1. When set to 1 onnx is built in debug mode. or debug versions of the dependencies, you need to open the [CMakeLists file][CMakeLists] and append a letter `d` at the end of the package name lines. For example, `NAMES protobuf-lite` would become `NAMES protobuf-lited`.  
+**Default**: Debug=0
 
-If `ONNX_USE_PROTOBUF_SHARED_LIBS` is ON then `Protobuf_USE_STATIC_LIBS` must be OFF and `USE_MSVC_STATIC_RUNTIME` must be 0.
-If `ONNX_USE_PROTOBUF_SHARED_LIBS` is OFF then `Protobuf_USE_STATIC_LIBS` must be ON and `USE_MSVC_STATIC_RUNTIME` can be 1 or 0.
+**CMake variables**
+* `ONNX_USE_PROTOBUF_SHARED_LIBS` should be ON or OFF. If `ONNX_USE_PROTOBUF_SHARED_LIBS` is ON then `USE_MSVC_STATIC_RUNTIME` must be 0. If `ONNX_USE_PROTOBUF_SHARED_LIBS` is OFF then `USE_MSVC_STATIC_RUNTIME` can be 1 or 0.  
```

Comment:
Can you expand this section to include the interaction between the ONNX_USE_PROTOBUF_SHARED_LIBS setting and the two (three if you count the one about MSVC STATIC RUNTIME) relevant options from protobuf?